sklearn/neighbors/approximate.py

```diff
@@ -189,11 +189,11 @@ class LSHForest(BaseEstimator, KNeighborsMixin, RadiusNeighborsMixin):
 
       >>> X_train = [[5, 5, 2], [21, 5, 5], [1, 1, 1], [8, 9, 1], [6, 10, 2]]
       >>> X_test = [[9, 1, 6], [3, 1, 10], [7, 10, 3]]
-      >>> lshf = LSHForest()
+      >>> lshf = LSHForest(random_state=42)
       >>> lshf.fit(X_train)  # doctest: +NORMALIZE_WHITESPACE
       LSHForest(min_hash_match=4, n_candidates=50, n_estimators=10,
```

Comment:
> Could you please cut down n_candidates and n_estimators to something more reasonable? Perhaps you'll need to lower min_hash_match to get any results.

Not an expert on LSHForest I am afraid. Using n_candidates=2, n_estimators=3, I get this:
```
distances = array([[ 0.0693931 ,  0.14960959],
           [ 0.4810007 ,  0.49225586],
           [ 0.00489444,  0.01897224]])

indices = array([[1, 2],
           [0, 1],
           [4, 3]])
```

Does that look more reasonable to you?